lint/lint_private.go

```diff
@@ -193,3 +194,17 @@ func addChecker(c abstractChecker, attrs ...checkerAttribute) {
 	}
 	checkerPrototypes[rule.name] = proto
 }
+
+func resolvePkgObjects(ctx *context, f *ast.File) {
+	ctx.pkgObjects = make(map[*types.PkgName]string)
```

Comment:
`make(map[*types.PkgName]string, len(f.Imports))` ?
Just to make it explicit what we're expecting, unless you're against this.

Reply:
OK.